code/modules/assembly/flash.dm

```diff
@@ -195,6 +194,12 @@
   * * attacker - Attacker
   */
 /obj/item/assembly/flash/proc/calculate_deviation(mob/victim, atom/attacker)
+	// Tactical combat emote-spinning should not counter intended gameplay mechanics.
+	// This trumps same-loc checks to discourage floor spinning in general to counter flashes.
+	// In short, combat spinning is silly and you should feel silly for doing it.
+	if(victim.flags_1 & IS_SPINNING_1)
+		return DEVIATION_NONE
```

Comment:
IMO, if you really have to nerf this, you should make this force a DEVIATION_PARTIAL, not a DEVIATION_NONE.

Reply:
We have conducted a survery of 1 ordinary members of the maintainer team. Your answer: "The spin emote should potentially provide some combat benefit against flashes, albeit minimal or situational" - Let's look at the board!

![tenor](https://user-images.githubusercontent.com/24975989/93632455-434a6f00-f9e5-11ea-8b86-79986fe90b41.gif)

If some other maintainer believes we absolute have to keep this in, they're welcome to come along and overrule me. But unless they do, I still stand by what I said.

I do not want to encourage rampant emote spam as a combat technique. As long as you have a convenient hotkey, or a keyboard that supports macros, or autohotkey, or a cat which responds well to metronomes, you can give yourself a combat advantage by just having it press your spin hotkey at the appropriate intervals, even if/after #53805 goes in.

Reply:
Well, it just seems really weird to me that spinning suddenly makes you vulnerable to being flashed. IMO, if we're using "you're spinning fast enough to somehow still catch the flash in your vision", we should be using the approximate average/median of the deviations for facing in each possible direction of the spin: DEVIATION_PARTIAL.

Reply:
> 
> 
> Well, it just seems really weird to me that spinning suddenly makes you vulnerable to being flashed. IMO, if we're using "you're spinning fast enough to somehow still catch the flash in your vision", we should be using the approximate average/median of the deviations for facing in each possible direction of the spin: DEVIATION_PARTIAL.
> 
> Like, DEVIATION_PARTIAL flashing still absolutely bones you about as much as DEVIATION_NONE flashing does, so combat spinning would still be still less effective than just trying to face away from the person flashing you manually (because at least with trying to face away from people manually, you have a chance to not get hardstunned).

The issue is that as long as you're spinning, you can still perform actions as if you were facing the target. The spin then immediately turns you away from them. There isn't a single situation when two people are having a flash battle where you would not want to spin.

I'm making the mother of all omlettes here, Jack. Can't fret over every egg.

This PR implements a simple solution to a simple problem. The problem is combat spinning gives benefits and the solution is that realism (what an argument when this entire PR is about people spinning in circles during fights to gain an advantage) gives way to gameplay.

Reply:
We shouldn’t even entertain a way to take on flashes as an aggressor. The best and only strategy should be staying the fuck away if you don’t have glasses (hopefully someone makes those less of a hard counter in the future).

Reply:
> The issue is that as long as you're spinning, you can still perform actions as if you were facing the target. The spin then immediately turns you away from them. There isn't a single situation when two people are having a flash battle where you would not want to spin.

Ohhhhhhhhh, okay, then yeah, that's much stronger than I thought it was.

Reply:
I'd also like to reiterate here that getting DEVIATION_PARTIAL flashed is still a gg, as you can just immediately be flashed twice more while you're hardstunned, no?